dsgrid/config/project_config.py

```diff
@@ -149,6 +154,7 @@ class InputDatasetModel(DSGBaseModel):
         title="model_sector",
```

Comment:
Do the todos above still require addressing? If so, do we have a plan for that? Should we discuss now and decide on next steps?

Reply:
I'll defer to @mooneyme